src/Engine/MapController.java

```diff
@@ -467,14 +467,14 @@ public void onStateChange()
         break;
       case DAMAGE_CHART:
         // Pull out the first enemy available, or ourselves
-        Commander targetCO = myGame.activeCO;
-        for( Commander co : myGame.commanders )
-          if( myGame.activeCO.isEnemy(co) )
+        Commander targetCO = myGame.activeArmy.cos[0];
+        for( Army army : myGame.armies )
+          if( myGame.activeArmy.isEnemy(army) )
           {
-            targetCO = co;
+            targetCO = army.cos[0];
```

Comment:
I'm thinking the currently active one would just be the first one. Just seems convenient.
As for picking opponents to compare to... agreed, if stat changes were accounted for. This is just the base damage chart, though.

Reply:
> I'm thinking the currently active one would just be the first one.

Are you saying `Army` will shuffle the `co`s collection each turn so that index 0 is always the active one? Because well, it currently doesn't, and I don't think I want it to.

That said, it doesn't matter whether we actually get the active one if the damage chart doesn't take CO differences into account. Though that calls into question whether we should include the damage chart at all if it's going to be misleading.

Reply:
Yes.
You are correct that it currently does not - our current only tag paradigm doesn't have an "active CO".
Why don't you want it to? It seems weird to me that the first CO in your list would not be the one that's currently active (e.g. I'm in a cart-tag with Grit and Max).

IMO it's valuable to know the base percentages, and I am planning another GUI to do detailed damage matchups between COs - it's just not a thing yet, because we have like 5 pending PRs and other priorities.
An option would be to make the attacker/defender a fixed color rather than matching your own, but then labeling the axes becomes more necessary.

Reply:
I mean, it could, but then you have extra logic, and the added complexity that `army.cos[0]` is not always the same CO. Not to mention that if it's the other army's turn, none of the COs would be active.

It feels like an overloading of responsibilities. `Army.cos` is a list of the COs in the army. If you want to know if or who is the active CO, I'd probably expect a separate accessor for that (which could return null if there is no such commander).

Reply:
> I mean, it could, but then you have extra logic, and the added complexity that `army.cos[0]` is not always the same CO.

We're gonna have to have logic that switches which CO is currently active regardless. It's a little bit more effort to shift the array contents around, but I think having a single source of truth is worth it.
As far as I can tell, `army.cos[0]` being the active CO instead of always the same CO is just a flat benefit, given our current codebase.
> Not to mention that if it's the other army's turn, none of the COs would be active.

Not at all true - the active CO is still who determines the D2D in cart tags and the one who gets full charging regardless.
> It feels like an overloading of responsibilities.

It is. I had exactly the same thoughts. And then I thought more - I believe it doesn't add any realized complexity for our use cases.
If you have a counterexample, I would like to hear it before I run into it myself.
> `Army.cos` is a list of the COs in the army. If you want to know if or who is the active CO, I'd probably expect a separate accessor for that (which could return null if there is no such commander).

That's probably fair, though it'd be a convenience accessor in all currently-planned cases - you can derive whether there's an active CO by the current tag mode, and you'd know what the active CO is (if any) based on CO list ordering.

Reply:
> Not at all true - the active CO is still who determines the D2D in cart tags and the one who gets full charging regardless

That is certainly a different definition for "active" than I had in mind. Not to mention that with team merges, you'd arguably have multiple "active" COs anyway.

I mean hey, if you feel strongly about it, I don't care too much if you want to implement CO shuffling. I think it's cleaner and simpler to use an accessor instead, but I'm not the one doing the work. I assume that's not something you plan to do in this PR (on account of not being relevant until we have a tag mode)?

Reply:
> That is certainly a different definition for "active" than I had in mind.

What were you talking about, then?

> I assume that's not something you plan to do in this PR (on account of not being relevant until we have a tag mode)?

Quite correct.

Reply:
Active. As in, currently acting. As in, the one currently able to move units. There can only be one of those at a time. I can see how your definition also makes sense though, in which case there would always be an active CO in each army, which is convenient. Still might have to sort the issue of having multiple active COs for team merges, but that's a problem for the future.